lib/config/env.ts

```diff
@@ -65,6 +68,14 @@ export function getConfig(env: NodeJS.ProcessEnv): RenovateConfig {
     });
   }
 
+  if (env.GITLAB_COM_TOKEN) {
+    config.hostRules.push({
+      hostType: PLATFORM_TYPE_GITLAB,
+      domainName: 'gitlab.com',
+      token: env.GITLAB_COM_TOKEN,
+    });
+  }
+
```

Comment:
@rarkins  Do we really want new env vars? Can't we just force users to use `hostRule` instead?

Reply:
Agreed, let's take a pause on adding new env vars and document that people should configure using `hostRules`. Sorry for any confusion

Reply:
@rubencabrera Please remove that